### PR TITLE
Add Supersole as speaker (pending)

### PR DESCRIPTION
**Pull request by auchenberg**

### CODE REVIEW

File: 2016/speakers.json [hunk 1]
```diff
@@ -48,10 +48,12 @@
...
-		"name": "To be announced",
-		"bio": "",
-		"avatar": "place5.png",
-		"placeholder": true
+		"name": "Soledad Penadés",
+		"slug": "soledad-penades",
+		"bio": "Sole has been building realtime stuff for longer than she can remember. She works at Mozilla, helping making the Web the best platform (in general) and the best platform for realtime (in particular). She writes demos to test and spread awareness about new features, often involving Web Audio, WebGL, GIFs and/or cats, and breaks various browsers and reports an unspeakable number of strange edge cases in the process, all in pursuit of a robust platform that is a joy to develop for.",
+		"avatar": "soledad-penades.png",
+		"twitter" : "supersole",
+		"github" : "supersole"
```
Comment (sole) [inline 2016/speakers.json]: sole in github :-)

**Comment by sole** [comment]
Other than the github fix, everything else is cool! Thanks
